doc/develop/modules.rst

```diff
@@ -493,6 +493,18 @@ Here is an example for the Zephyr module ``foo``:
    use ``bar`` as its module name if nothing is specified in
    :file:`zephyr/module.yml`.
 
+
+Module version
+==============
+
+Each Zephyr module should have a version number to improve tracking.
```

Comment:
`should` seems a bit strong in this case, especially as it would indicate that modules are having some form of release cycles. This may be true for some modules, but other modules revision might be considered to be identified (thus indirectly follow the Zephyr version) through the SHA in the manifest, which again is identified by the Zephyr revision / version.

```suggestion
Each Zephyr module may have a version number to improve tracking.
```